### PR TITLE
Colm/apb 1251

### PR DESCRIPTION
**Pull request by cavster**

### CODE REVIEW

File: it/support/AppAndStubs.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  override lazy val port: Int = Port.randomAvailable
+
+  override implicit lazy val app = FakeApplication(
+    additionalConfiguration = additionalConfiguration
+  )
+
+  protected def additionalConfiguration: Map[String, Any] = {
+    Map(
+      "microservice.services.auth.host" -> wireMockHost,
```
Comment (arturopala) [inline it/support/AppAndStubs.scala]: this `host` setting is redundant, we should either configure it for every service or better leave default

File: it/support/StartAndStopWireMock.scala [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  override def beforeAll() = {
+    wireMockServer.stop()
+    wireMockServer.start()
+    WireMock.configureFor(wireMockHost, wireMockPort)
+  }
+
+  override def beforeEach() = {
+    WireMock.reset()
+  }
```
Comment (arturopala) [inline it/support/StartAndStopWireMock.scala]: here was previously audit stub setting, it should be now included in every controller test?

Comment (cavster) [reply it/support/StartAndStopWireMock.scala]: If you think it's better in there I will add it back in?

Comment (arturopala) [reply it/support/StartAndStopWireMock.scala]: It doesn't fit in this place after split, shall be added to the tests

Comment (cavster) [reply it/support/StartAndStopWireMock.scala]: Awesome

**Pull request merged**